### PR TITLE
Various clean up

### PR DESCRIPTION
**Pull request by mikepitre**
left aligned all elements on the meetings show page,
changed "invite already sent" to "invite sent!"
changed invite table name to "invite employees to this meeting"
added location of meeting to headline

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -86,9 +86,9 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/rooms">
+      <a class="navbar-brand" href="/">
         <% if company_logo == nil %>
-          <a class="navbar-brand" href="/rooms">CRS2000</a>
+          <a class="navbar-brand" href="/">CRS2000</a>
```
Comment (jah2488) [inline app/views/layouts/application.html.erb]: Could we change this to be a `link_to` tag that goes to the `root_path` instead?

File: app/views/meetings/show.html.erb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 <div>
     <div class="col-md-12">
-      <h2><%= @meeting.title.capitalize %></h2>
+      <h2><%= @meeting.title.titleize %> in <%= @meeting.room.name %></h2>
```
Comment (jah2488) [inline app/views/meetings/show.html.erb]: Shouldn't we add `titleize`to `room.name` as well?

**Comment by jah2488** [comment]
:tada:

File: app/views/meetings/show.html.erb [hunk 3]
```diff
@@ -70,7 +70,7 @@
             <tr>
               <td>
                 <% if invitee.invitations.where(meeting_id: @meeting.id).exists? %>
-                    <br><p style="font-style: italic">Already invited!</p>
+                    <br><p style="font-style: italic">Invite Sent!</p>
```
Comment (mlgiardina) [inline app/views/meetings/show.html.erb:67]: i already changed this in the merge i just did

File: app/views/meetings/show.html.erb (hunk 2 above)
Comment (mikepitre) [inline app/views/meetings/show.html.erb]: done!

**Comment by mlgiardina** [comment]
:+1:

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (mikepitre) [inline app/views/layouts/application.html.erb]: Done!

**Pull request merged**